src/mqt/predictor/driver.py

```diff
@@ -5,10 +5,14 @@ def compile(qc, model="ML", opt_objective="fidelity"):
     if model == "ML":
         predictor = ml.Predictor()
         prediction = predictor.predict(qc)
-        return predictor.compile_predicted_compilation_path(qc, prediction)
+        compiled_qc = predictor.compile_predicted_compilation_path(qc, prediction)
+        compile_information = ml.helper.get_index_to_comppath_LUT()[prediction]
+        return compiled_qc, compile_information
+
     elif model == "RL":
         predictor = rl.Predictor()
-        return predictor.compile(qc, opt_objective=opt_objective)
+        compiled_qc, compile_information = predictor.compile(qc, opt_objective)
+        return compiled_qc, compile_information
```

Comment:
It would really be great if both predictors could just share the same interface.
I like the way how the RL predictor just calls `compile` and straight up returns the circuit and the information.
Would you mind refactoring the ML part so that it shares the same interface?
Then the `if` here could look something like
```python
if model == "ML":
  return ml.Predictor().compile(...)
elif model == "RL":
  return rl.Predictor().compile(...)
```

And now that I think of it, an even cleaner interface would be to simply have functions `ml.compile` and `rl.compile` that are wrappers around constructing a `Predictor` object and calling the respective function.

Reply:
Implemented now.